docs/templates/sso-cd-x509-https.adoc

```diff
@@ -166,7 +166,7 @@ for more information.
 |`JGROUPS_PING_PROTOCOL` | -- | openshift.DNS_PING
 |`OPENSHIFT_DNS_PING_SERVICE_NAME` | -- | `${APPLICATION_NAME}-ping`
 |`OPENSHIFT_DNS_PING_SERVICE_PORT` | -- | 8888
-|X509_CA_BUNDLE | -- | `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`
+|X509_CA_BUNDLE | A set of CA certificates | `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`
```

Comment:
This is the same location listed twice. Should the second one be changed to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`?

Reply:
Yet, manual template docs change like this one will be lost when the `gen_template_docs.py` tool is run next time. I would rather have this change described in a separate file (see `sso.adoc.in` file for example of such a thing), and have the `gen_template_docs.py` tool modified, so it inserts the expected content into the template docs each time it is run.

Reply:
After thinking about this a bit longer, I think I should remove the description. Otherwise, I would need to put the `X509_CA_BUNDLE` into the template parameters, which adds only noise to the big picture. Let me just rerun the `gen_template_docs.py` and leave it as is.

Reply:
> This is the same location listed twice. Should the second one be changed to /var/run/secrets/kubernetes.io/serviceaccount/ca.crt

Of course, you are correct!